swap_meet/item.py

```diff
@@ -1,2 +1,19 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = None, condition = 0):
+        self.category = category if category is not None else ""
+        self.condition = condition if condition is not None else 0
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        describers = ["You might want gloves for this one..",
+        "It could look worse", 
+        "Average wear and tear",
+        "This looks almost new!",
+        "This is NEW!"
+        ]
+
+        for item in range(len(describers) + 1):
+            if item == self.condition:
```

Comment:
You could also create a dictionary and and set the condition value as the key and the description as the key and access the description like this: `return descriptions[self.condition]`